src/constants.ts

```diff
@@ -15,9 +15,9 @@ export const TEAM_COLORS = [
 
 export const TEAM = [
   {
-    CoinIcon: TeamOneCoin
+    CoinIcon: TeamOneCoinF
   },
   {
-    CoinIcon: TeamTwoCoin
+    CoinIcon: TeamTwoCoinD
```

Comment:
@Da-Colon I was going to add logic for these based on DAO names or something but would rather just keep it simple and hardcode these, I'll swap for the E on day 2